eng/common/TestResources/New-TestResources.ps1

```diff
@@ -159,7 +166,14 @@ try {
 
     # Enumerate test resources to deploy. Fail if none found.
     $repositoryRoot = "$PSScriptRoot/../../.." | Resolve-Path
-    $root = [System.IO.Path]::Combine($repositoryRoot, "sdk", $ServiceDirectory) | Resolve-Path
+
+    $root = ''
+    if (Split-Path -IsAbsolute $ServiceDirectory) {
+        $root = $ServiceDirectory
+    } else {
+        $root = [System.IO.Path]::Combine($repositoryRoot, "sdk", $ServiceDirectory) | Resolve-Path
+    }
```

Comment:
This is unnecessary. `Path.Combine` will use the last path element if it's absolute already:

![image](https://user-images.githubusercontent.com/1532486/135350332-3046ebac-e765-4c1c-acb4-73354a5c48d3.png)